src/main/java/org/dita/dost/platform/FileGenerator.java

```diff
@@ -88,7 +108,7 @@ public void generate(final File fileName) {
         } catch (final RuntimeException e) {
             throw e;
         } catch (final Exception e) {
-            logger.error("Failed to transform " + fileName + ": " + e.getMessage(), e);
+            throw new IOException("Failed to transform " + fileName + ": " + e.getMessage(), e);
```

Comment:
If generating a file from a template fails, it's likely that the actual DITA-OT process will not work. So it's safer to fail to notify that this doesn't work, instead of logging and continuing.